lib/rubocop/cop/rails/attribute_default_block_value.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for `attribute` class methods that specify a `:default` option
+      # and pass it a method without a block.
```

Comment:
Can you add a description and examples for allowed patterns such as literals and constants to the doc?

Reply:
Great idea!

Description and examples added and I updated the .adoc file.